internal/olm/installer/client.go

```diff
@@ -229,6 +229,10 @@ func (c Client) doRequest(ctx context.Context, url string) (*http.Response, erro
 		defer resp.Body.Close()
 		body, err := ioutil.ReadAll(resp.Body)
 		msg := fmt.Sprintf("failed GET '%s': unexpected status code %d, expected %d", url, resp.StatusCode, http.StatusOK)
+		if resp.StatusCode == 404 {
+			noManifestsMsg := fmt.Sprintf("OLM version %s tag is present in GitHub, but install manifests cannot be located", version)
+			return nil, fmt.Errorf("%s: %s", msg, noManifestsMsg)
```

Comment:
This isn't necessarily true, since 404 just means the requested resource was not found; 404 is returned if a tag does not exist too. This should read something like:
```suggestion
			return nil, fmt.Errorf("%s; manifests may not exist for this release, please check the releases page for olm.yaml and crds.yaml", msg)
```